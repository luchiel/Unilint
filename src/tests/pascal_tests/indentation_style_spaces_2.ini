[indentation]
indentation_style = spaces
indentation_policy = by_size
indentation_size = 2
