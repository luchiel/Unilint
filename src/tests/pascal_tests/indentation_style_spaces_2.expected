2:1: tabs and spaces mixed
8:1: indent too small
10:1: indent too deep
11:1: spaces expected, tabs found
